components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -228,7 +228,7 @@ const collectivePickerSearchQuery = gqlV1/* GraphQL */ `
         isArchived
         isHost
         payoutMethods {
-          id
+          legacyId: id
```

Comment:
I prefer that we avoid that and make sure to use `legacyId` instead of `id` where it's required.